cmake/optional.cmake

```diff
@@ -189,12 +135,13 @@ else()
 endif()
 
 if(WITH_GUI AND WITH_QRENCODE)
-  if(MSVC)
+  if(VCPKG_TARGET_TRIPLET)
```

Comment:
Why was this changed?

Reply:
This makes an intention clearer because the logic is specific to vcpkg package manager, not to the MSVC compiler.

Reply:
Is anyone using MSVC without vcpkg? Is this being done consistently?

Reply:
> Is anyone using MSVC without vcpkg?

I don't think so. But vcpkg might be used on other OSes.

> Is this being done consistently?

I hope it is consistent now in the staging branch.